go/libkb/globals.go

```
@@ -238,6 +238,7 @@ func (g *GlobalContext) PushShutdownHook(sh ShutdownHook) {
 }
 
 func (g *GlobalContext) ConfigureConfig() error {
+	RemoteSettingsRepairman(g)
```

is it OK to ignore this error?